pkg/controller/metrics.go

```diff
@@ -23,20 +23,20 @@ import (
 	"go.opencensus.io/stats/view"
 )
 
-const metricPrefix = observability.MetricRoot + "/server/login"
-
-var (
-	MFirebaseRecreates = stats.Int64(metricPrefix+"/fb_recreates", "firebase user recreates", stats.UnitDimensionless)
```

Comment:
This appears to be an old, unused metric, so I removed it.

Reply:
It looks like the behavior was removed in: https://github.com/google/exposure-notifications-verification-server/pull/902

Can we leave the metric - I think we want to instead keep that functionality

Reply:
We have no associated alert for the metric. I can add it back in, but do you also want to create an alert for it?

Reply:
https://github.com/google/exposure-notifications-verification-server/pull/1607 to bring it back.

We don't need an alert for it. I wanted it there to see (later) if this was a common problem, but it allows users to get unstuck in this situation